webapp/src/scriptsearch.tsx

```diff
@@ -269,6 +280,10 @@ export class ScriptSearch extends data.Component<ISettingsProps, ScriptSearchSta
         const urldata = this.fetchUrlData();
         const local = this.fetchLocal()
 
+        const isSearchingGh = ghdata.status === data.FetchStatus.Pending || ghdata.status === data.FetchStatus.Error;
```

Comment:
So if it errors is it just going to show the spinner forever?

Reply:
Well, in `data.ts`, errors are re-queued immediately, so that's why I made it that way... but that means if the request keeps getting errors, it will indeed spin forever.

I guess I can change errors to mean "request completed", and if the re-queued request happens to succeed, then the results will be silently added to the search results. Not a big deal, since transient request errors should be very rare